src/outlet/ticketStorage.ts

```diff
@@ -402,4 +402,19 @@ export class TicketStorage {
 	private storeTickets() {
 		localStorage.setItem(TicketStorage.LOCAL_STORAGE_KEY, JSON.stringify(this.ticketCollections))
 	}
+
+	public migrateLegacyTokenStorage(issuerConfig: OffChainTokenConfig, tokenStorageKey = 'dcTokens') {
+		if (!issuerConfig || !tokenStorageKey) {
+			errorHandler('Issuer config and token storage key are required for migration.', 'error', null, null, true, false)
+		}
+		const storageData = localStorage.getItem(tokenStorageKey)
+		if (storageData) {
+			let ticketCollectionsUpdated = JSON.parse(localStorage.getItem(TicketStorage.LOCAL_STORAGE_KEY)) as unknown as TicketStorageSchema
+			const createCollectionHashes = createIssuerHashArray(issuerConfig)
```

Comment:
@nicktaras You need to create the hash for the actual attestation, not the issuerConfig, which may include several hashes.

Reply:
Let's review over a call or dm thread to confirm the solution details, this implementation is inspired from other parts of the library such as ticketZKProof which using the same method to create the issuerHash array.

// line 59, ticketZKProof
const createCollectionHashes = createIssuerHashArray(issuerConfig)